### ISSUE TITLE
Unify naming of `bmmmodels`

### ISSUE THREAD

**Issue by GidonFrischkorn** [venpopov/bmm#134]
After we started working on the M3 models on Friday, I noticed that we might want to established a unified naming scheme for the different models. At least have some conventions so that there is a system how different versions and specific version of them are labelled and called.

For example, for the Interference Measurement Model (IMM) all models are labelled `IMMversion` that is `IMMabc`, `IMMbsc`, and `IMMfull`. To follow this we decided to label the M3 models accordingly, also `M3version`, that is `M3ss`(simple span), `M3cs` (complex span), and `M3custom` for setting up a custom M3 model.

If we were to take this as a convention, we could think about renaming the Signal Discrimination Model to `SDMsimple`. Accordingly, additional versions would be labelled `SDMfull`.

What do you think about this?

**Comment by venpopov** [comment]
I've been thinking about the same in the last week. Let's discuss this in person. One other option I was thinking about is consolidation. E.g., we have only one function IMM that takes a version argument.

```
IMM <- function(resp_error, nt_features, nt_distances = NULL, version = "full") {
## code here
cl = c('bmmmodel", "vwm", "non_targets")
class(out) = c(cl, paste0('IMM', version))
out
} 
```
Not sure yet whether that makes it easier or harder for the code

**Comment by venpopov** [comment]
Closed by #178
